cypress/integration/chi-vue/data-table.spec.js

```diff
@@ -471,85 +470,132 @@ describe('Data Table', () => {
       });
 
       it(`should sort by name with the correct icons`, () => {
-        cy.get('@firstRow').should('contain', 'Name 1');
+        cy.get('@rows')
+          .eq(0)
+          .should('contain', 'Name 1');
         cy.get(`[data-cy='data-table-sorting'] .${DATA_TABLE_CLASSES.ROW}`)
           .eq(0)
           .find(`.${DATA_TABLE_CLASSES.CELL}`)
           .eq(0)
           .as('firstCell')
-          .find('.chi-icon')
+          .find(`.${ICON_CLASS}`)
           .as('sortIcon');
-        hasClassAssertion('@sortIcon', 'icon-arrow-sort');
+        hasClassAssertion('@sortIcon', `${ARROW_SORT_CLASS}`);
         cy.get('@firstCell')
           .click()
           .then(() => {
-            hasClassAssertion('@sortIcon', 'icon-arrow-up');
-            cy.get('@firstRow').should('contain', 'Name 1');
+            hasClassAssertion('@sortIcon', `${ARROW_UP_CLASS}`);
+            cy.get('@rows')
+              .eq(0)
+              .should('contain', 'Name 1');
             cy.get('@sortIcon').should('have.css', 'transform', 'none');
           });
         cy.get('@firstCell')
           .click()
           .then(() => {
-            hasClassAssertion('@sortIcon', 'icon-arrow-up');
+            hasClassAssertion('@sortIcon', `${ARROW_UP_CLASS}`);
             cy.get('@sortIcon')
               .should('have.attr', 'style')
               .should('contain', 'transform: rotate(180deg);');
-            cy.get('@firstRow').should('contain', 'Surname 2');
+            cy.get('@rows')
+              .eq(0)
+              .should('contain', 'Surname 2');
           });
         cy.get('@firstCell')
           .click()
           .then(() => {
-            cy.get('@firstRow').should('contain', 'Name 1');
-            hasClassAssertion('@sortIcon', 'icon-arrow-sort');
+            cy.get('@rows')
+              .eq(0)
+              .should('contain', 'Name 1');
+            hasClassAssertion('@sortIcon', `${ARROW_SORT_CLASS}`);
           });
       });
 
       it(`should sort by status using a custom template`, () => {
-        cy.get('@firstRow').should('contain', 'Name 1');
+        const indexes = [0, 1, 2];
+        indexes.forEach(index => {
+          if (index % 2 === 0) {
```

Comment:
Is the case of having the first 2 rows inactive and the third active?
In that case, Could you simplify the if statement? `if (index === 2)`?